### PR TITLE
NicoJeskeIT.MasterVolumeSync version 1.0.7

### PR DESCRIPTION
**Pull request by nicojeske**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97643)

### CODE REVIEW

**Comment by nicojeske** [comment]
@microsoft-github-policy-service agree

File: manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NicoJeskeIT.MasterVolumeSync
+PackageVersion: 1.0.7
+PackageLocale: en-US
+Publisher: Nico Jeske IT
+PackageName: Master Volume Sync
```
Comment (mdanish-kh) [inline manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.locale.en-US.yaml:8]:
```suggestion
PackageName: MasterVolumeSync
PackageUrl: https://github.com/nicojeske/MasterVolumeSync
```
Should match Control Panel / Apps&Features / Registry's `DisplayName` key as WinGet relies on the metadata from registry for matching upgrades.

Also suggested additional metadata.

File: manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NicoJeskeIT.MasterVolumeSync
+PackageVersion: 1.0.7
+PackageLocale: en-US
+Publisher: Nico Jeske IT
```
Comment (mdanish-kh) [inline manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.locale.en-US.yaml:7]:
```suggestion
Publisher: Nico Jeske IT
PublisherUrl: https://nicojeske.de/
PublisherSupportUrl: https://github.com/nicojeske/MasterVolumeSync/issues
```
Additional metadata

File: manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: NicoJeskeIT.MasterVolumeSync
+PackageVersion: 1.0.7
+PackageLocale: en-US
+Publisher: Nico Jeske IT
+PackageName: Master Volume Sync
+License: MIT License
```
Comment (mdanish-kh) [inline manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.locale.en-US.yaml:9]:
```suggestion
License: MIT License
LicenseUrl: https://github.com/nicojeske/MasterVolumeSync/blob/master/LICENSE.MD
Copyright: Copyright (c) 2022 Nico Jeske
CopyrightUrl: https://github.com/nicojeske/MasterVolumeSync/blob/master/LICENSE.MD
ReleaseNotes: |-
  - Muting the selected output device will now also mute the "What u hear" device. (#3)
ReleaseNotesUrl: https://github.com/nicojeske/MasterVolumeSync/releases/tag/1.0.7
```
Additional metadata

File: manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: NicoJeskeIT.MasterVolumeSync
+PackageVersion: 1.0.7
```
Comment (mdanish-kh) [inline manifests/n/NicoJeskeIT/MasterVolumeSync/1.0.7/NicoJeskeIT.MasterVolumeSync.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.7
ReleaseDate: 2022-06-25
```
Additional metadata

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Note: The metadata comments are optional for you to include but they would make for a more complete manifest.

**Comment by nicojeske** [comment]
Thank you, @mdanish-kh, for your review. Your feedback is greatly appreciated. I must admit that I was not aware of all the metadata fields that you pointed out. I had assumed that wingetcreate would inform me about possible additional metadata.

I did end up removing the PublisherURL field, as the pipeline did seem to have problems connecting to the site (403 forbidden). This access seems to be blocked by Cloudflare, as my internal monitoring shows no 403 response from the website.

**Comment by mdanish-kh** [comment]
> I had assumed that wingetcreate would inform me about possible additional metadata.

wingetcreate is designed to let publishers quickly create and submit a manifest with the required fields. If you wish to have a detailed manifest, you could use another manifest authoring tool supported by the WinGet team [YamlCreate.ps1](https://github.com/microsoft/winget-pkgs/blob/master/doc/tools/YamlCreate.md).
 
> I did end up removing the PublisherURL field, as the pipeline did seem to have problems connecting to the site (403 forbidden). This access seems to be blocked by Cloudflare, as my internal monitoring shows no 403 response from the website.

In that case, you could update it to link to your GitHub profile if you wish to.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**